README.md

```diff
@@ -343,17 +341,18 @@ This is how we create and publish a new release of the 121-platform.
 
 ### Patch/Hotfix Checklist
 
-This follows the same process as a regular release + deployment. With some small changes.
+This follows a similar process to regular release + deployment, with some small changes.
 
-- Code does not need to be frozen. (As there is no active development on the release-branch)
-
-- Checkout the `release/<version>`-branch that needs the hotfix.
-- Create a new local branch on top of it (e.g. `release/<v1.x.1>`) and make the changes
+- Code does not need to be frozen.
+- Checkout the `<version>` tag which contains the code that you want to hotfix from.
+- Create a new local hotfix branch using that tag as the head (e.g. `hotfix/<v1.x.1>`) and make the changes
 - Add the hotfix-release to the [CHANGELOG](CHANGELOG.md)
 - Push this branch to the upstream/origin repository.
-- Create a new release (see above) and publish it.
+- Create a new release (see above) selecting the `hotfix` branch as target, and publish it.
 - Use the [deployment-workflows on GitHub Actions](https://github.com/global-121/121-platform/actions) to deploy to production (for each instance)
-- After the hotfix-release, apply/include the fix by creating a PR from the hotfix-branch to `main`-branch
+- After the hotfix has been released to production, follow standard procedures to merge the hotfix branch into main
+
+**Note:** it is important that you do not rebase the `hotfix` branch on the head of `main` until AFTER you have successfully deployed the hotfix to production. The hotfix branch was created from a "dangling" commit, so this makes the GitHub UI confused when you look at a PR between the newly created `hotfix` branch and `main`, but you can ignore any conflict warnings from GitHub while you release.
```

Comment:
I guess the tag for a hotfix would also be 'dangling', I wonder what happens to it when the hotfix-branch gets deleted (when merging the PR)...?

It would be nice that the "compare links" between all the different versions mentioned in the CHANGELOG will remain working. And that they show the exact specific commits that are relevant... (and not any weird double-commits or anything)

Reply:
I'm pretty sure that the commit still remains valid, it simply won't be attached to a specific branch. So the compare links should still work. (I'm not 100% confident though).

Reply:
Lets see. :)